sdk/types/tsconfig.json

```diff
@@ -6,9 +6,10 @@
         "esModuleInterop": true,
         "sourceMap": true,
         "declaration": true,
-        "outDir": "dist"
+        "outDir": "dist",
+        "strict": true
     },
     "include": [
-        "gen/**/*"
+        "src/**/*"
```

Comment:
this should be gen

Reply:
Ah yes, I was trying to fix the fact that `types.input.ts` was erroring because it wasn't being included by `tsconfig.json`.

I fixed this by creating a separate `tsconfig.json` in `src` and not emitting "noEmit": true`.

I added `tsc --project src` to `package.json build` so it is still type-checked.

https://github.com/koush/scrypted/pull/1308/commits/e595d0a55b7b9db45615a13b947feba4aeb2ebf9